x-pack/plugins/security_solution/public/common/lib/triggers_actions_ui/register_alerts_table_configuration.tsx

```diff
@@ -17,6 +17,9 @@ const registerAlertsTableConfiguration = (
   registry: AlertsTableConfigurationRegistryContract,
   storage: Storage
 ) => {
+  if (registry.has(APP_ID)) {
```

Comment:
Do we have the proper unit test for this fix? I think we need to add the test coverage here.